cypress/integration/pages/articles/testsForCanonicalOnly.js

```diff
@@ -91,6 +91,39 @@ export const testsThatFollowSmokeTestConfigForCanonicalOnly = ({
           });
         });
 
+        if (service === 'news') {
+          it('should render a visible guidance message', () => {
+            cy.window().then(win => {
+              const aresMedia = getBlockData(
+                'video',
+                win.SIMORGH_DATA.pageData,
+              );
+              const longGuidanceWarning =
+                aresMedia.model.blocks[1].model.blocks[0].model.versions[0]
+                  .warnings.long;
+
+              cy.get('div[class^="StyledVideoContainer"]')
+                .eq(0)
+                .within(() => {
+                  // Check for video with guidance message
+                  if (longGuidanceWarning) {
+                    cy.get('div[class^="StyledGuidance"]')
+                      .should('to.have.descendants', 'svg')
+                      .within(() => {
+                        cy.get('strong');
+                      })
+                      .should('be.visible')
+                      .should('not.be.empty')
+                      .and('contain', longGuidanceWarning);
```

Comment:
When these methods on L115-117 are chained from the `.within()` block instead of the `cy.get('strong')` itself are we sure they're being run on the correct element?

Reply:
Yes, @jamesbrumpton and I confirmed this yesterday